index.html

```diff
@@ -10,24 +11,42 @@
     <div id="home-screen">
         <div id="home-content">
             <h1>Auber Space Protector</h1>
-            <a href="#page-content"><button type="button" class="btn btn-primary">Documentation</button></a>
-            <a href="" download><button type="button" class="btn btn-success">Download</button></a>
+            <div>
+                <a href="#page-content"><button type="button" class="btn btn-primary">Documentation</button></a>
```

Comment:
It seems weird to me that this doesn't lead directly to the documentation

Reply:
Where does it lead? All I did was move the links from the other page to the main page. When I click on them it takes to the the same place as before. Is that not right?

Reply:
That's my mistake. Its no meant to.  I've renamed it to 'Documents  and Links'